src/iap/layout/components/layout-header/layout-header.component.html

```diff
@@ -0,0 +1,58 @@
+<div class="iap-header">
```

Comment:
:point_right: :point_left: 
```suggestion
<div class="iap-layout-header">
```